src/Drupal/Driver/Fields/Drupal8/FileHandler.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\Driver\Fields\Drupal8;
+
+/**
+ * File field handler for Drupal 8.
+ */
+class FileHandler extends EntityReferenceHandler {
```

Comment:
It is not strictly necessary, but that is all that is needed to support file fields now that we have entity support. I will split it into its own PR if you prefer.